config/locales/views.fr.yml

```
@@ -70,6 +70,7 @@ fr:
       date_creation: Date de création
       date_radiation: Date de radiation
       forme_juridique: Forme juridique
+      information: Informations sur l'entreprise
```

```suggestion
      information: Informations sur l’entreprise
```
Pour perpétuer les habitudes de Nico 😄   